.gitignore

```diff
@@ -54,4 +54,7 @@ robots.txt
 .vscode
 
 # Crowdin report output
-src/data/crowdin/bucketsAwaitingReviewReport.csv
\ No newline at end of file
+src/data/crowdin/bucketsAwaitingReviewReport.csv
+
+# Storybook
+build-storybook.log
```

Comment:
We might also need to ignore `/storybook-static` since it is generated when you run the tests locally.